docs/docs/native-docker.md

```diff
@@ -1,17 +1,69 @@
 
-Marathon enables users to run Docker container images with two different runtimes:
+Marathon enables users to launch containers with container images using two different runtimes:
 
-  1. [Docker containerizer](# Docker Containerizer) using the native Docker Engine as runtime.
-  2. Mesos containerizer using the Universal Container Runtime.
+1. Mesos containerizer using the [Universal Container Runtime](#ucr).
+1. [Docker containerizer](#docker-containerizer) using the native Docker Engine as runtime.
 
-# Docker Containerizer
+<a name="ucr"></a>
+
+# Universal Container Runtime
+
+The [Universal Container Runtime](http://mesos.apache.org/documentation/latest/container-image) (UCR) extends the Mesos containerizer to support provisioning [Docker](https://docker.com/) container images ([AppC](https://github.com/appc/spec) coming soon). This means that you can use both the Mesos containerizer and other container image types. You can still use the Docker container runtime directly ([instructions are below](#docker-containerizer)), but the Universal Container Runtime supports running Docker images without depending on the Docker Engine, which allows for better integration with Mesos.
+
+The following Marathon features _only_ work with the UCR:
+
+- [Pods]({{ site.baseurl }}/docs/pods.html).
+- GPUs.
+- [Authentication to a private Docker registry using a secret store]({{ site.baseurl }}/docs/native-docker-private-registry.html).
+
+## Provisioning Containers with the UCR
+
+To provision containers with the UCR, specify the container type `MESOS` and a the appropriate object in your application definition. Here, we specify a Docker container with the `docker` object.
+
+The UCR containerizer provides a `pullConfig` parameter with a `secret` field for [authentication with a private Docker registry]({{ site.baseurl }}/docs/native-docker-private-registry.html). 
+
+`credential`, with a `principal` and an optional `secret` field to authenticate when downloading the Docker image.
+
+```json
+{  
+   "id":"mesos-docker",
+   "container":{  
+      "docker":{  
+         "image":"mesosphere/inky",
+         "pullConfig": {
+                "secret": "pullConfigSecret"
+          }
+      },
+      "type":"MESOS"
+   },
+   "secrets": {
+        "pullConfigSecret": {
+            "source": "/mesos-docker/pullConfig"
+        }
+   "args":[  
+      "<my-arg>"
+   ],
+   "cpus":0.2,
+   "mem":16.0,
+   "instances":1
+}
+```
+
+**Important:** If you leave the `args` field empty, the default entry point will be the launch command for the container. If your container does not have a default entry point, you must specify a command in the `args` field. If you do not, your app will fail to deploy.
+
+## UCR Limitations
+- The UCR does not support the following: runtime privileges, Docker options, force pull, named ports, numbered ports, bridge networking, port mapping.
```

Comment:
AFAIK we're shipping support for bridge networking and port mappings in Marathon 1.5.

I'm also pretty sure that Marathon supports forced image pull for MESOS containers using a docker image. Is the limitation mentioned here due to a mesos implementation bug? @Gilbert88 

Not sure how "numbered ports" or "named ports" made this list either... should probably be removed. Those are Marathon'ish things, not Mesos things.